utils/contract-deployment/README.md

```diff
@@ -17,4 +17,21 @@ OR
 `go run utils/contract-deployment/contractDeploymentTools.go deriveContractAddress 0x38545c4b331D8BFb3bee94C62D77a6735b5eF8c0 1`
 
 ## Results
-The resulting raw transaction, `TeleporterMessenger` contract address, and universal deployer address are written to standard output, as well as to `UniversalTeleporterDeployerTransaction.txt`, `UniversalTeleporterMessengerContractAddress.txt`, and `UniversalTeleporterDeployerAddress.txt` respectively.
\ No newline at end of file
+The resulting raw transaction, `TeleporterMessenger` contract address, and universal deployer address are written to standard output, as well as to `UniversalTeleporterDeployerTransaction.txt`, `UniversalTeleporterMessengerContractAddress.txt`, and `UniversalTeleporterDeployerAddress.txt` respectively.
+
+## Deploy the contract
+Now that the keyless transaction is constructed, fund the deployer address. For example, using `cast`:
+
+```bash
+teleporter_deploy_tx=$(cat UniversalTeleporterDeployerTransaction.txt)
```

Comment:
I think this should be reading in `$teleporter_deploy_address`, not the transaction